mlrun/config.py

```diff
@@ -325,6 +325,11 @@
         "requests": {"cpu": None, "memory": None, "gpu": None},
         "limits": {"cpu": None, "memory": None, "gpu": None},
     },
+    # preemptible node selector labels and tolerations to be added when running on spot nodes
+    "preemptible_nodes": {
+        "node_selector_labels": "e30=",
```

Comment:
Selector is a very known term in the k8s world, I think the world labels is redundant, and again, best to stick to what we did in Nuclio (`nodeSelector`)
If you really feel like `labels` are needed then I would go with `node_labels_selector` (it's the selector of the labels of the node)